include/IGPUBuffer.h

```diff
@@ -8,6 +8,7 @@
 
 #include "irr/core/IBuffer.h"
 #include "IDriverMemoryBacked.h"
+#include "irr/asset/IDescriptor.h"

```

Comment:
you probably have an uncomitted file, can't find it in the tree

Reply:
@Crisspl done?

Reply:
y